packages/sit-onyx/src/components/OnyxPagination/OnyxInlinePagination.vue

```diff
@@ -0,0 +1,207 @@
+
+  <div
+    v-else
+    :class="[
+      'onyx-component',
+      'onyx-pagination',
+      'onyx-pagination__inline',
+      'onyx-text',
+      densityClass,
+    ]"
+    role="group"
+    :aria-label="t('pagination.label')"
+    :style="{ '--onyx-pagination-character-count': props.modelValue.toString().length }"
+  >
+    <button
+      class="onyx-pagination__inline__button"
+      :aria-label="t('pagination.previous')"
+      type="button"
+      :disabled="props.disabled || hasReachedMin"
+      @click="emit('update:modelValue', props.modelValue - 1)"
+    >
+      <OnyxIcon :icon="iconChevronLeftSmall" />
+    </button>
+    <template v-for="pageNumber in displayPagesNumbers" :key="`page-${pageNumber}`">
+      <button
+        v-if="typeof pageNumber === 'number'"
+        :class="[
+          'onyx-pagination__inline__button',
+          { 'onyx-pagination__inline__button--active': pageNumber === props.modelValue },
+        ]"
+        :aria-label="t('pagination.buttonLabel', { page: pageNumber })"
+        type="button"
+        :disabled="props.disabled"
+        @click="emit('update:modelValue', pageNumber)"
+      >
+        {{ pageNumber }}
+      </button>
+      <div
+        v-else
+        :class="['onyx-pagination__inline__ellipsis']"
+        :aria-label="t('pagination.ellipsis')"
+      >
+        <span aria-hidden="true">&hellip;</span>
+        <OnyxVisuallyHidden>
+          {{ t("pagination.ellipsis") }}
+        </OnyxVisuallyHidden>
+      </div>
+    </template>
+
+    <button
+      class="onyx-pagination__inline__button"
+      :aria-label="t('pagination.next')"
+      type="button"
+      :disabled="props.disabled || hasReachedMax"
+      @click="emit('update:modelValue', props.modelValue + 1)"
+    >
+      <OnyxIcon :icon="iconChevronRightSmall" />
+    </button>
+  </div>
+</template>
+
+<style lang="scss">
+@use "../../styles/mixins/layers.scss";
+
+.onyx-pagination__inline {
+  @include layers.component() {
+    background-color: var(--onyx-color-base-background-blank);
+
+    &__button,
+    &__ellipsis {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      padding: var(--onyx-pagination-padding-vertical);
+      height: var(--onyx-pagination-height);
+      border: var(--onyx-pagination-border-size) solid var(--onyx-color-component-border-neutral);
+      border-left: none;
+      background-color: inherit;
+      color: inherit;
+      aspect-ratio: 1;
+    }
+    &__ellipsis {
+      cursor: default;
+    }
+    &__button {
+      cursor: pointer;
+      color: var(--onyx-color-text-icons-neutral-medium);
+
+      &:first-of-type {
+        border-radius: var(--onyx-radius-sm) 0 0 var(--onyx-radius-sm);
+        border-left: var(--onyx-pagination-border-size) solid
+          var(--onyx-color-component-border-neutral);
+      }
+      &:last-of-type {
+        border-radius: 0 var(--onyx-radius-sm) var(--onyx-radius-sm) 0;
```

Comment:
```suggestion
        border-radius: var(--onyx-pagination-border-radius) 0 0 var(--onyx-pagination-border-radius);
        border-left: var(--onyx-pagination-border-size) solid
          var(--onyx-color-component-border-neutral);
      }
      &:last-of-type {
        border-radius: 0 var(--onyx-pagination-border-radius) var(--onyx-pagination-border-radius) 0;
```

Since we use the border radius more than 1 time, I'd prefer to add a new CSS variable for this so its easier to adjust the value in the future